.github/workflows/pipeline.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.8', '3.9', '3.10']
```

Comment:
Why do we need to drop 3.11?

Reply:
Dependency resolving of 3.11 is really slow. Seems to be related to both `kfp` and `google-cloud-aiplatform`